lib/PhasarLLVM/DataFlow/IfdsIde/Problems/IDEGeneralizedLCA/ConstantHelper.cpp

```diff
@@ -24,9 +25,8 @@ bool psr::glca::isConstant(const llvm::Value *Val) {
   if (llvm::isa<llvm::ConstantPointerNull>(Val)) { // NULL
     return true;
   }
-  if (const auto *Gep = llvm::dyn_cast<llvm::ConstantExpr>(Val);
-      Gep && Val->getType()->isPointerTy() &&
-      Val->getType()->getPointerElementType()->isIntegerTy()) {
+  if (const auto *Gep = llvm::dyn_cast<llvm::GEPOperator>(Val);
+      Gep && Gep->getResultElementType()->isIntegerTy()) {

```

Comment:
Gep != nullptr

Reply:
The code in the if-condition is already long enough, so I don't really want to add the `!= nullptr` to it. However, If you feel that's important for readability, I will add it